fuse/new_orleans_pd.py

```diff
@@ -32,7 +32,15 @@ def fuse_cprr(cprr, actions, officer_number_dict):
     )
     # cprr.loc[:, 'action'] = cprr.allegation_primary_key.map(
     #     lambda x: actions_dict.get(x, ''))
-    return rearrange_allegation_columns(cprr)
+    return cprr
+
+
+def fuse_pib(pib, officer_number_dict):
+    pib = float_to_int_str(cprr, ["officer_primary_key"])
+    pib.loc[:, "uid"] = cprr.officer_primary_key.map(
+        lambda x: officer_number_dict.get(x, "")
+    )
```

Comment:
`cprr` needs to be passed in explicitly as a function argument.

Reply:
Thanks